kqueen/engines/openstack_kubespray.py

```diff
@@ -196,7 +199,14 @@ def provision(self):
         try:
             self.cluster.state = config.CLUSTER_PROVISIONING_STATE
             self.cluster.save()
-            app.executor.submit(self._run_provisioning)
+            future = app.executor.submit(self._run_provisioning)
```

Comment:
future? imo its not a good naming)

Reply:
that is how object called, that ThreadPoolExecutor returns https://docs.python.org/3/library/concurrent.futures.html#executor-objects